src/components/Tool/FixPhoneNumbers/Contact.test.tsx

```diff
@@ -118,6 +123,7 @@ describe('FixPhoneNumbers-Contact', () => {
             handleAdd={handleAddMock}
             handleChangePrimary={handleChangePrimaryMock}
             setContactFocus={setContactFocus}
+            handleUpdate={updatePhoneNumber}
```

Comment:
Maybe you could test that this mock function gets called in this`should call mock functions` test block.